client/src/main/com/sinch/sdk/domains/voice/models/svaml/InstructionSetCookie.java

```diff
@@ -1,20 +1,22 @@
 package com.sinch.sdk.domains.voice.models.svaml;
 
+import com.sinch.sdk.core.models.OptionalValue;
+
 public class InstructionSetCookie extends Instruction {
 
-  private final String key;
-  private final String value;
+  private final OptionalValue<String> key;
+  private final OptionalValue<String> value;
```

Comment:
The documentation doesn't specify these properties must not be null, but does it make sense to set a property while the other is not defined?

Reply:
Right 👍 
Fixed (but, by purpose,  let possibility to end user setting a cookie named 'null' ....)